src/components/Pressable/utils.ts

```diff
@@ -146,7 +192,7 @@ const innerStyleKeys = new Set([
   'paddingVertical',
   'start',
   'end',
-  'direction', // iOS only
+  'direction', // IOS only
```

Comment:
This should be left as `iOS` 😅 

```suggestion
  'direction', // iOS only
```